app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -67,6 +67,17 @@ def forgot_password_new_password_submitted(success:, failure_reason: nil)
       )
     end
 
+    # @param [String] decision One of 'reject' or 'pass'
+    # @param [String] fraud_fingerprint An opaque identifier to correlate with a TMX check
+    # A profile offlined for review has been approved or rejected.
+    def fraud_review_adjudicated(decision:, fraud_fingerprint:)
+      track_event(
+        :fraud_review_adjudicated,
+        decision: decision,
+        fraud_fingerprint: fraud_fingerprint,
+        )
```

Comment:
spacing/alignment
```suggestion
      )
```

Reply:
Strongly agree. Two thoughts:

1. I just switched editors, from IntelliJ with Ruby plugins to RubyMine. I imported the old settings, but it seems to have taken the opportunity to get... creative... with formatting.
2. Should there be rubocop policies for this stuff? I've now pushed up a few commits on this where bizarre indentation got snuck in and rubocop has let it go.

Reply:
Ohhhh lulz, it _did_ flag it in this case. 🤦